homeassistant/components/statistics/sensor.py

```diff
@@ -203,6 +208,13 @@ def extra_state_attributes(self):
                 ATTR_CHANGE_RATE: self.change_rate,
             }
 
+        return {
+            ATTR_SAMPLING_SIZE: self._sampling_size,
+            ATTR_COUNT: self.count,
```

Comment:
We normally don't duplicate values in the state and state attributes.

Reply:
Firstly, I'm content to remove this if that's preferred and nobody wants to wait for me to push a 2nd rev and just have a maintainer make the edit before merging I have no objection.

To explain why I chose to include it initially, though: It was added is because the `ATTR_MEAN: self.mean,` is returned in the other form of this sensor (which is also the state value). For derived templates it's convenient to have the same attributes have the same meaning for the same sensor type. Since my goal was to try to map as closely as possible to the attributes available this sensor is operating on other input it seemed worth duplicating to achieve that.

Reply:
I agree that it's not optimal to represent different kinds of measurements with the same entity state. On the other hand the user defines the source samples so should know what type of result is measured.

Reply:
Sorry, it didn't post my comment when I resolved.

I agree with your point and I'm pushing a new rev. If it were being built again it would probably omit `ATTR_MEAN` and adding a duplicate attribute for the sake of parity is more tech debt.